lms/templates/courseware_search/search_list.underscore

```diff
@@ -10,11 +10,11 @@
     <% if (hasMoreResults) { %>
         <a class="search-load-next" href="javascript:void(0);">
             <%= interpolate(
-                ngettext("Load next %(num_items) result", "Load next %(num_items) results", pageSize),
+                ngettext("Load next %(num_items) result", "Load next %(num_items)s results", pageSize),
```

Comment:
Does the 's' character fix something here - if so, then you'll need it in the first string as well.